bots/zephyr_mirror_backend.py

```diff
@@ -368,8 +382,7 @@ def process_notice(notice, log):
     if notice.format.startswith("Zephyr error: See") or notice.format.endswith("@(@color(blue))"):
         logger.debug("Skipping message we got from Zulip!")
         return
-    if (zephyr_class == "mail" and notice.instance.lower() == "inbox" and is_personal and
-            not options.forward_mail_zephyrs):
+    if (zephyr_class == "mail" and notice.instance.lower() == "inbox" and is_personal and not options.forward_mail_zephyrs):
```

Comment:
As above---for this PR I prefer that @anirudhjain75 not make any changes except those specifically adding type annotations, which is why I told him to return these lines to the way they were originally.